lib/languages.ts

```diff
@@ -582,6 +582,17 @@ const definitions: Record<LanguageKey, LanguageDefinition> = {
         previewFilter: null,
         monacoDisassembly: null,
     },
+    snowball: {
+        name: 'Snowball',
+        monaco: 'rust',
```

Comment:
Is `rust` really the right choice here?

Reply:
I believe is the one who works better with snowball. I can't think of any better, sorry.

Reply:
Ok: most languages will define their own syntax highlighting: if you're happy with `rust` then so be it :)

Reply:
Im trying to search for other highlightings...

Reply:
im keeping with `rust` 👍🏽. It covers for most of the functionality (it can work for now)